servicelayer/archive/file.py

```diff
@@ -50,3 +50,33 @@ def archive_file(self, file_path, content_hash=None, mime_type=None):
 
     def load_file(self, content_hash, file_name=None, temp_path=None):
         return self._locate_key(content_hash)
+
+    def store_export(self, namespace, file_path, mime_type=None):
```

Comment:
Do we actually want to support this for `file` archives? That sort of pre-supposes a) that we implement a serving API in Aleph, b) that the generated URLs will be publishable (which is not true for intranet deployments of Aleph).

Reply:
Not useful in production but I think it's still useful in development setup to test run the export dashboard setup?